<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AwesomeViewControls</class>
 <widget class="QWidget" name="AwesomeViewControls">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>336</width>
    <height>712</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Awesome View</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="selectImageLabel">
     <property name="styleSheet">
      <string notr="true">QLabel { color: rgb(255, 0, 0) }</string>
     </property>
     <property name="text">
      <string>Please select an image.</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QFormLayout" name="PatientInfolayout">
     <property name="horizontalSpacing">
      <number>0</number>
     </property>
     <property name="verticalSpacing">
      <number>15</number>
     </property>
     <property name="leftMargin">
      <number>20</number>
     </property>
     <property name="topMargin">
      <number>20</number>
     </property>
     <property name="bottomMargin">
      <number>20</number>
     </property>
     <item row="0" column="0">
      <widget class="QLabel" name="Profil_title_label">
       <property name="text">
        <string>Profile</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_8">
       <property name="text">
        <string>Name:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLabel" name="PatientName_label">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_11">
       <property name="text">
        <string>Gender:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLabel" name="PatientGender_label">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_9">
       <property name="text">
        <string>Date of birth:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLabel" name="PatientDOB_label">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_10">
       <property name="text">
        <string>Patient ID:</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLabel" name="PatientID_label">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="pixelValue">
       <property name="text">
        <string>PixelValue(s) (Left Click + Shift):</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPushButton" name="paintButton">
     <property name="text">
      <string>Paint Tool</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTableWidget" name="m_LabelSetTableWidget"/>
   </item>
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Images:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QListWidget" name="listWidget"/>
   </item>
   <item>
    <widget class="QCheckBox" name="m_VolRenderingButton">
     <property name="text">
      <string>Enable Volume Rendering</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="STLsExport_button">
     <property name="text">
      <string>Export STLs</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="pbExportCSV">
     <property name="text">
      <string>CSV Export</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="pushButtonOtsuSegmentation">
     <property name="text">
      <string>Otsu Segmentation</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>75</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
